src/lightning_app/core/constants.py

```diff
@@ -5,6 +5,26 @@
 
 import lightning_app
 
+
+def truthy(val) -> bool:
```

Comment:
Where is this used in this PR?

Reply:
this was a leak from my PR division. I'll add it to the right PR. Thank you